src/Tizen.NUI.Components/Controls/ScrollableBase.cs

```diff
@@ -14,12 +14,13 @@
  *
  */
 using System;
-using Tizen.NUI.BaseComponents;
 using System.Collections.Generic;
 using System.ComponentModel;
 using System.Diagnostics;
 using System.Runtime.InteropServices;
+
 using Tizen.NUI.Accessibility;
+using Tizen.NUI.BaseComponents;
```

Comment:
are Classes in BaseComponents used in ScrollableBase?

Reply:
View is in BaseComponents